cmd/nerdctl/login_linux_test.go

```diff
@@ -78,6 +78,7 @@ func TestLoginWithSpecificRegHosts(t *testing.T) {
 }
 
 func TestLoginWithPlainHttp(t *testing.T) {
+	t.Parallel()
```

Comment:
The port number will conflict with other PRs

Reply:
removed 👍🏻